index.js

```diff
@@ -302,6 +303,58 @@ p.setRoomData = function( roomID, data ) {
 };
 
 /**
+ * Set room as public
+ * 
+ * @param  {String} roomID id for the room you'd like to make public
+ * @return {Promise} This promise will succeed when the room as been set as public
+ */
+p.setRoomPublic = function( roomID ) {
+
+	if( roomData[ roomID ] === undefined ) {
+		
+		return promise.reject( 'There is no room by that id' );
+	} else {
+		publicRooms.push( roomID );
+		return promise.resolve( );
```

Comment:
I think these pull requests across all repos look great. I wonder if there is something meaningful we could return when these promises resolve. Same applied in `setRoomPrivate`